app/renderer/js/pages/preference/find-accounts.js

```diff
@@ -0,0 +1,79 @@
+'use-strict';
+
+const { remote, ipcRenderer } = require('electron');
+
+const { dialog } = remote;
+
+const BaseComponent = require(__dirname + '/../../components/base.js');
+
+class FindAccounts extends BaseComponent {
+	constructor(props) {
+		super();
+		this.props = props;
+	}
+
+	template() {
+		return `
+			<div class="settings-card certificate-card">
+				<div class="certificate-input">
+					<div>Organization URL</div>
+					<input class="setting-input-value" value="zulipchat.com"/>
+				</div>
+				<div class="certificate-input">
+					<button class="green w-150" id="find-accounts-button">Find accounts</button>
+				</div>
+			</div>
+		`;
+	}
+
+	init() {
+		this.$findAccounts = this.generateNodeFromTemplate(this.template());
+		this.props.$root.appendChild(this.$findAccounts);
+		this.$findAccountsButton = this.$findAccounts.querySelector('#find-accounts-button');
+		this.$serverUrlField = this.$findAccounts.querySelectorAll('input.setting-input-value')[0];
+		this.initListeners();
+	}
+
+	findAccounts(url) {
+		if (!url) {
+			return;
+		}
+		if (!url.includes(':')) {
```

Comment:
Checking for `:` is not a good logic to determine the presence of prefix. Check the url starts with `http`. `:` can be included while specifying ports which is usually a case in self-hosted servers so that is not a good check.

Reply:
Hmm, I didn't think of that. Thanks, I'll change it.